### DISCUSSION TITLE
Support Pakajo As Shipping Carrier

### DISCUSSION THREAD

**Discussion by Nikolasgrizli** [etsy/open-api#1338] [Q&A]
Hello Etsy developer team and community!

We would like to request to add "Pakajo" to the list of carriers that can be trackable on Etsy.

As Pakajo, we are the tech based shipping and fulfillment provider widely used by Etsy sellers in Germany and worldwide.
We offer shipping services and also partner with Ebay, and we want to integrate with ETSY for seller orders, our customers use this order data to create shipping label on Pakajo.world. We also work with Aftership.
We would like to be a carrier partner in ETSY sellers for their user experience. I'm talking about this text from your documentation:
 
„Etsy supports tracking updates for several carriers, determined by the carrier_name submitted with a createShippingReceipt request. Tracking updates provide a tracking link in the shipping notification email and displays tracking updates for the shipment on Etsy.“
 
I found some discussions about this here and saw that you can do this if you have enough information.
 
To help us understand each other better, I would like to give some examples of how we relate to a service like Aftership:
 
Example for UPG-Trackingno.: UPG-281338, UPG281338, 281338
 
XML Output:
https://media-sc.com:65312/10/trackingdaten/auflisten.xml?api_user=aftership&api_key=c08e433c62cbcfe98fdfa917be443ef7&id_sendungsdaten=UPG-281338&harmonisierte_codes=1&flag_intermediate_label=1 
https://media-sc.com:65312/10/trackingdaten/auflisten.xml?api_user=aftership&api_key=c08e433c62cbcfe98fdfa917be443ef7&id_sendungsdaten=UPG281338&harmonisierte_codes=1&flag_intermediate_label=1
https://media-sc.com:65312/10/trackingdaten/auflisten.xml?api_user=aftership&api_key=c08e433c62cbcfe98fdfa917be443ef7&id_sendungsdaten=281338&harmonisierte_codes=1&flag_intermediate_label=1
 
 
JSON Output:
https://media-sc.com:65312/10/trackingdaten/auflisten.json?api_user=aftership&api_key=c08e433c62cbcfe98fdfa917be443ef7&id_sendungsdaten=UPG-281338&harmonisierte_codes=1&flag_intermediate_label=1
https://media-sc.com:65312/10/trackingdaten/auflisten.json?api_user=aftership&api_key=c08e433c62cbcfe98fdfa917be443ef7&id_sendungsdaten=UPG281338&harmonisierte_codes=1&flag_intermediate_label=1
https://media-sc.com:65312/10/trackingdaten/auflisten.json?api_user=aftership&api_key=c08e433c62cbcfe98fdfa917be443ef7&id_sendungsdaten=281338&harmonisierte_codes=1&flag_intermediate_label=1
 
Event-Codes:
//  10: Data Transmitted
//  20: Dist. Center(DE)
//  30: Logistic Center
//  40: In Delivery
//  50: Delivered
// 500: Delivery failed
// 600: Parcel Returned
 
Aftership Integration:
https://www.aftership.com/carriers/pakajo
 
Example of Aftership Tracking:
https://www.aftership.com/track?t=281338&c=pakajo
 
If you have any further questions or require additional information, please do not hesitate to reach out. Thank you in advance for your assistance.

P.S.: Yes, I know this is more of a place for the community that uses the Etsy api, but I've been trying to find some way to get Etsy to add our tracking link for customers for about six months and hope this can help)) I wrote to the dev team a couple of times and unfortunately they replied that they don't have a team to create a new connection right now

**Comment by kiernangs** [comment]
Let  me know if you ever discover a way to communicate with etsy to request a adding shipping carrier support.

After the one employee monitoring get hub left I have never been able to find a way to get them to add carriers (emailing support, emailing developers, etc)  nothing seems to work.

Good luck!

I would love to have cirro Ecommerce added (https://www.cirrotrack.com/)  = http://aftership.com/track?c=parcll

But after 4 years of requests im not holding my breath any more

**Comment by kiernangs** [comment]
Turns out... after years of asking specifically if they had any form for requesting adding support for a new carrier (as recently as last week)  they did  have such a request form all along. They just didnt tell their support people.

I finally today talked to a support person named jack who sent me this etsy link for requesting carriers:  https://survey.alchemer.com/s3/7650632/Etsy-Feedback-Form-jaf267ho6w
